src/SAF.DAS.ApprenticeCommitments.Web.UnitTests/Features/MyApprenticeshipOverviewSteps.cs

```diff
@@ -19,10 +19,12 @@ public class MyApprenticeOverviewSteps : StepsBase
         private readonly TestContext _context;
         private readonly RegisteredUserContext _userContext;
         private HashedId _apprenticeshipId;
-        private bool _EmployerConf, _TrainingProviderConf, _ApprenticeshipDetailsConf, _RolesAndResponsibilitiesConf, _HowApprenticeshipWillBeDeliveredConf;
+        private bool _EmployerConf, _TrainingProviderConf, _ApprenticeshipDetailsConf, _RolesAndResponsibilitiesConf, _HowApprenticeshipWillBeDeliveredConf, _IsStopped;
         private DateTime _confirmationDeadline;
         private ChangeOfCircumstanceNotifications _changeOfCircumstanceNotifications;
         private long _revisionId;
+        private DateTime? _StoppedReceivedOn = null;
```

Comment:
should really be _stopped....

Reply:
its been given the same name as in the dto - StoppedReceivedOn is the datetime, IsStopped is the bool

Reply:
private vars should not be capitalised